src/controllers/extension-limit-reached.controller.ts

```diff
@@ -0,0 +1,27 @@
+import {NextFunction, Request, Response} from "express";
```

Comment:
filename doesn't match standard of using dots instead of hyphens

Reply:
chaged over to dots unit test as well.